samples/csharp_dotnetcore/50.teams-messaging-extensions-search/TeamsMessagingExtensionsSearch.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="AdaptiveCards" Version="2.5.1" />
```

Comment:
Please remove the reference to AdaptiveCards library, since it is only used for the AdaptiveCard.ContentType. The sample can just hard code 'application/vnd.microsoft.card.adaptive'